package.json

```diff
@@ -87,33 +87,47 @@
     }
   },
   "scripts": {
-    "test": "npm run build && playwright test",
-    "test:debug": "DEBUG=pw:browser* npm run build && playwright test",
+    "test": "pnpm build && playwright test",
+    "test:debug": "DEBUG=pw:browser* pnpm build && playwright test",
     "rollup:preload": "rollup -c rollup.preload.config.ts --configPlugin @rollup/plugin-typescript --bundleConfigAsCjs",
     "rollup:main": "rollup -c rollup.main.config.ts --configPlugin @rollup/plugin-typescript --bundleConfigAsCjs",
-    "build": "npm run rollup:preload && npm run rollup:main",
-    "start": "npm run build && electron ./dist/index.js",
-    "start:debug": "ELECTRON_ENABLE_LOGGING=1 npm run start",
+    "build": "pnpm rollup:preload && pnpm rollup:main",
+    "start": "pnpm build && electron ./dist/index.js",
+    "start:debug": "ELECTRON_ENABLE_LOGGING=1 pnpm start",
     "postinstall": "patch-package",
     "clean": "del-cli dist && del-cli pack",
-    "dist": "npm run clean && npm run build && electron-builder --win --mac --linux -p never",
-    "dist:linux": "npm run clean && npm run build && electron-builder --linux -p never",
-    "dist:mac": "npm run clean && npm run build && electron-builder --mac dmg:x64 -p never",
-    "dist:mac:arm64": "npm run clean && npm run build && electron-builder --mac dmg:arm64 -p never",
-    "dist:win": "npm run clean && npm run build && electron-builder --win -p never",
-    "dist:win:x64": "npm run clean && npm run build && electron-builder --win nsis-web:x64 -p never",
+    "dist": "pnpm clean && pnpm build && electron-builder --win --mac --linux -p never",
+    "dist:linux": "pnpm clean && pnpm build && electron-builder --linux -p never",
+    "dist:mac": "pnpm clean && pnpm build && electron-builder --mac dmg:x64 -p never",
+    "dist:mac:arm64": "pnpm clean && pnpm build && electron-builder --mac dmg:arm64 -p never",
+    "dist:win": "pnpm clean && pnpm build && electron-builder --win -p never",
+    "dist:win:x64": "pnpm clean && pnpm build && electron-builder --win nsis-web:x64 -p never",
     "lint": "eslint .",
     "changelog": "auto-changelog",
-    "release:linux": "npm run clean && npm run build && electron-builder --linux -p always -c.snap.publish=github",
-    "release:mac": "npm run clean && npm run build && electron-builder --mac -p always",
-    "release:win": "npm run clean && npm run build && electron-builder --win -p always",
+    "release:linux": "pnpm clean && pnpm build && electron-builder --linux -p always -c.snap.publish=github",
+    "release:mac": "pnpm clean && pnpm build && electron-builder --mac -p always",
+    "release:win": "pnpm clean && pnpm build && electron-builder --win -p always",
     "typecheck": "tsc -p tsconfig.json --noEmit"
   },
```

Comment:
Is this really a needed change?

Personally I like it, but this kinda forces the user that wants to build from source w/o intentions of contributing to use pnpm.

One example would be the [AUR package for youtube-music](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=youtube-music), this is a breaking change.

`npm run build` works fine for us pnpm users, but the reverse does not hold true.

Reply:
`pnpm build` still works w/o the script changing is what I mean

Reply:
This package uses the `pnpm patch` (formerly `patch-package` in npm) feature, I think it should rely on pnpm.

Reply:
oh, yeah then its ok

Reply:
https://github.com/NixOS/nixpkgs/pull/260995

I found another package for `nixOS` that depends on `npm`, so I will hold off on changing it